config/rbac/role.yaml

```diff
@@ -4,6 +4,13 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
```

Comment:
do we really need 'create' permissions?

Reply:
You're right, we don't. Dropped them, left only the 'get' permissions